Gruntfile.js

```diff
@@ -1,5 +1,11 @@
 module.exports = function(grunt) {
 
+    // Configuration options for Amazon S3 service
+    var awsConfig = {
```

Comment:
@leemunroe I was going to do that at first but decided against it for a reason I now forget. I've moved it to the secrets file. I blame it on late night coding :)